### ISSUE TITLE
Use more informative link texts

### ISSUE THREAD

**Issue by bencomp** [NLeSC/guide#173]
**Suggestion**

To improve accessibility and search engine processability, hyperlinks should have texts that explain what they link to. I found myself having to hover links often to see what is behind the *here* or other generic term.

I can make a start with some pages if you like.

**Comment by jiskattema** [comment]
Thank you for having a look at our guide, any readability improvements are welcome! If you create some pull requests, i will try to have them merged.

**Comment by bouweandela** [comment]
Done in #174
